src/components/CodeView/index.spec.tsx

```diff
@@ -100,9 +100,11 @@ describe(__filename, () => {
   };
 
   const renderWithMessages = ({
+    location = createFakeLocation(),
```

Comment:
This seems unnecessary as `location` is in `RenderParams`, but TS was mad about my use of `location` in the new tests where I am calling `renderWithMessages`. I think it's a case of TS getting confused by spreads, which I think we've seen in the past. Without adding this here, I was getting TS errors in the test code when trying to pass `location` into `renderWithMessages`.